pandas/io/parsers.py

```diff
@@ -225,6 +225,8 @@
     Indicate number of NA values placed in non-numeric columns.
 skip_blank_lines : bool, default True
     If True, skip over blank lines rather than interpreting as NaN values.
+skip_blank_columns: bool, default False
```

Comment:
if we end up doing this we will want to put this argument at the end because it's possible (if bad practice) to pass these arguments as positionals